web/src/lib/components/elements/buttons/circle-icon-button.svelte

```diff
@@ -1,19 +1,32 @@
+<script lang="ts" context="module">
+  export type Color = 'transparent' | 'light' | 'dark' | 'gray' | 'primary' | 'opaque';
+</script>
+
 <script lang="ts">
   import Icon from '$lib/components/elements/icon.svelte';
-  type Color = 'transparent' | 'light' | 'dark' | 'gray' | 'primary' | 'opaque';
 
   export let type: 'button' | 'submit' | 'reset' = 'button';
   export let icon: string;
   export let color: Color = 'transparent';
   export let title: string;
+  /**
+   * The padding of the button, used by the `p-{padding}` Tailwind CSS class.
```

Comment:
From what I understand, tailwind scans these source files and extracts used class names, which it includes in the final build of the app. Using a dynamic class name like this doesn't work. I mean it would be possible to refer to a class that won't actually be used. It probably happens to work because the same classes are already used and detected, but I think this is an anti-pattern.

Reply:
I will take a look at how the tailwind classes are generated in this scenario. If it is an anti-pattern, I'd propose changing that in a following PR to prevent increasing the scope of this change.

Reply:
I created a derived variable, so the class name ho longer has a class name with a dynamic number like that.